cogeo_mosaic/utils.py

```diff
@@ -237,7 +240,7 @@ def create_mosaic(
 
         maxzoom = max(maxzoom)
 
-    quadkey_zoom = minzoom
+    quadkey_zoom = quadkey_zoom or minzoom
```

Comment:
@kylebarron I think this is the only change between your PR and mine

Reply:
I think
```py
quadkey_zoom = quadkey_zoom or minzoom
```
is simpler than
```py
    if quadkey_zoom is None:
        quadkey_zoom = minzoom
```

Reply:
I know that's why I write it!